packages/fuselage/src/components/SideBarV2/SideBarFooter/SideBarFooter.tsx

```diff
@@ -0,0 +1,11 @@
+import type { HTMLAttributes } from 'react';
+import React from 'react';
+
+export const SideBarFooter = ({
+  children,
+  ...props
+}: HTMLAttributes<HTMLElement>) => (
+  <footer className='rcx-sidebar-v2-footer' {...props}>
```

Comment:
```suggestion
  <div className='rcx-sidebar-v2-footer' {...props}>
```

I **think** we shouldn't use this tag, we'll need to do some research

Reply:
I found some information saying you can have multiple footers on the same document, it represents the footer of a section - what you shouldn't have is multiple footers on a single section.
wdyt?

https://www.w3schools.com/tags/tag_footer.asp#:~:text=You%20can%20have%20several%20%3Cfooter%3E%20elements%20in%20one%20document.